server/events/project_command_runner.go

```diff
@@ -234,14 +236,29 @@ func (p *DefaultProjectCommandRunner) Plan(ctx command.ProjectContext) command.P
 // PolicyCheck evaluates policies defined with Rego for the project described by ctx.
 func (p *DefaultProjectCommandRunner) PolicyCheck(ctx command.ProjectContext) command.ProjectResult {
 	policySuccess, failure, err := p.doPolicyCheck(ctx)
+	var policyCheckApprovals []models.PolicySetApproval
```

Comment:
Not sure if this is the right place for calculating PolicySetApproval, but put it here to preserve structure of doPolicyCheck.